src/state/sync/UserAssetsSync.tsx

```diff
@@ -6,7 +6,7 @@ import { useSwapsStore } from '@/state/swaps/swapsStore';
 import { selectUserAssetsList, selectorFilterByUserChains } from '@/__swaps__/screens/Swap/resources/_selectors/assets';
 import { ParsedSearchAsset } from '@/__swaps__/types/assets';
 import { ChainId } from '@/__swaps__/types/chains';
-import { useUserAssets } from '../resources/assets';
+import { useUserAssets } from '@/__swaps__/screens/Swap/resources/assets';
```

Comment:
These have slightly different implementations and `config` options — just want to make sure that was considered.

Reply:
Errrr... I think this is still what it is on develop. I think because I moved it to the state/sync folder it looks odd though. On develop we're referencing the swaps `useUserAssets`: https://github.com/rainbow-me/rainbow/blob/208856b9d92db0f5346c02284603dd3afd108b1d/src/__swaps__/screens/Swap/components/UserAssetsSync.tsx#L9

Reply:
Ah sorry, mistook the change for switching to the other version of `useUserAssets`, missed that this file moved.